leafsmart-front-end/src/components/EventCard.js

```diff
@@ -1,26 +1,31 @@
 import React from "react";
+import moment from "moment";
 
 export default function EventCard(props) {
   const { name, image, date, url } = props;
 
+  const currentDate = moment(date).format('MMMM Do YYYY');
+
   return (
     <>
-      <tr>
-        <td class="border px-8 py-1">
-          <h1>{name}</h1>
+      <tr className="bg-white border-b transition duration-300 ease-in-out hover:bg-gray-100">
+        <td class="whitespace-nowrap text-sm font-medium">
           <img
-            className="h-auto w-28 pb-2/3 sm:pt-1/3"
+            className="h-auto w-64 px-6 py-4 whitespace-nowrap"
             src={image}
             alt={name}
           />
         </td>
-        <td class="border text-xs px-9 py-1">
-          <p>{date}</p>
+        <td classNam="text-sm text-Independence font-light px-2 py-2 whitespace-nowrap">
```

Comment:
typo here for "className" 👀

Reply:
Cheers this has been fixed and re-submitted